tests/src/rv32-scall.s

```diff
@@ -0,0 +1,10 @@
+# RISC-V syscall test
```

Comment:
I think it should be placed to "riscv-mars-examples", along with the Makefile.
We do not put assembly sources for tests to this repository, excepting kernel exception hanlders (which _are_ the part of the simulation).